mail.php

```diff
@@ -0,0 +1,66 @@
+<?php
+require 'vendor/autoload.php';
+require 'validation.php';
+use Dotenv\Dotenv;
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+/**
+ * A class used for sending email to recipient;
+ */
+class mail
+{
```

Comment:
Class names should always start with Capital letter.
```suggestion
class Mail {
```